test/docfx.Test/build/OpsConfigAdapterTest.cs

```diff
@@ -61,8 +60,7 @@ public static async Task AdaptOpsServiceConfigWithAAD(string url, string expecte
         {
             Skip.If(
                 bool.TryParse(Environment.GetEnvironmentVariable("IS_GITHUB_ACTION"), out var isGithubAction)
-                && isGithubAction
-                && !RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
+                && isGithubAction);
```

Comment:
We don't need to invent our own environment variables, See https://docs.github.com/en/actions/reference/environment-variables for a list of predefined environment variables on github predefined environment varables.

Reply:
@live1206 my bad, didn't find this.

Reply:
updated